Automattic/jetpack
# Boost: Partial move styles to their respective components

Part of https://github.com/Automattic/jetpack/issues/35010.

## Proposed changes:

* Add path aliases to make it easier to move styles around and import stylesheets/reach assets;
  * `$css` - alias for `./app/assets/src/css`. Example `@import "$css/main/mixins.scss";`;
  * `$images` alias for `./app/assets/static/images`. Example `$images/color-shift.svg`;
* Update stylesheets to use aliases;
* Move some styles to their respective components;
* Remove unused styles;
* Move upgrade markup into separate component;

### Other information:


## Jetpack product discussion



https://github.com/Automattic/jetpack/issues/35010

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Make sure the UI isn't broken.
